videotroncmdb.json

```diff
@@ -0,0 +1,290 @@
+{
+    "appid": "e9279bd9-6fbf-4aae-9982-83109d9e5423",
+    "name": "Videotron CMDB",
+    "description": "This app integrates with the Videotron CMDB tool to perform lookups",
+    "type": "information",
+    "product_vendor": "Videotron",
+    "logo": "logo_videotroncmdb.svg",
+    "logo_dark": "logo_videotroncmdb_dark.svg",
+    "product_name": "CMDB",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2020-2021 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2020-07-14T10:41:41.182339Z",
+    "package_name": "phantom_videotroncmdb",
```

Comment:
* Add  bs4, requests, and all the dependent modules in pip_dependencies and add wheel files as well.
* Add bs4 and requests libraries in requirements.txt
Refer Tala for the same.

Reply:
Added bs4 and request wheel files and dependent modules in pip_dependencies and updated requirements.txt for same.